stdlib/public/Cxx/std/String.swift

```diff
@@ -26,6 +30,14 @@ extension std.string: ExpressibleByStringLiteral {
 }
 
 extension String {
+  /// Creates a String having the same content as the given C++ string.
+  ///
+  /// If `cxxString` contains ill-formed UTF-8 code unit sequences, this
+  /// initializer replaces them with the Unicode replacement character
+  /// (`"\u{FFFD}"`).
+  ///
+  /// - Complexity: O(*n*), where *n* is the number of characters in the C++
+  ///   string.
   public init(cxxString: std.string) {
```

Comment:
I guess we plan to update this to a no label initializer right, based on today's discussion. Also regarding the documentation, should we say "number of bytes in the C++ string" instead of "number of characters"?

Reply:
Yes, I'll update the initializer in a separate patch.
Updated the doc comment, thanks!